src/components/Applications.jsx

```diff
@@ -50,7 +51,12 @@ const getApplicationsList = data => {
         !hiddenApps.includes(app.slug.toLowerCase())
     )
     const dedupapps = uniqBy(apps, 'slug')
-    const appList = dedupapps.map(app => <AppTile key={app.id} app={app} />)
+
+    const sortedApps = flag('apps.sort')
```

Comment:
is it a new flag? We should probably name it home.apps.sort or is it a global flag?

Reply:
It is a new flag, documented, that is indeed global (it is used in cozy-bar).